<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckForm</class>
 <widget class="QWidget" name="CheckForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проверить код подлинности</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Код подлинности</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCod"/>
     </item>
     <item>
      <widget class="QPushButton" name="cmdGo">
       <property name="toolTip">
        <string>Проверить код</string>
       </property>
       <property name="text">
        <string>Проверить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
